app/src/main/java/nu/yona/app/security/EncryptionUtils.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * <?xml version="1.0" encoding="utf-8"?><!--
+ * ~ Copyright (c) 2018 Stichting Yona Foundation
+ *   ~
+ *   ~ This Source Code Form is subject to the terms of the Mozilla Public
+ *   ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+ *   ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *   -->
+ */
+
+package nu.yona.app.security;
+
+import android.content.Context;
+import android.os.Build;
+
+import java.io.IOException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+
+import nu.yona.app.utils.AppUtils;
+
+
+public class EncryptionUtils
+{
+
+	public static String encrypt(Context context, String token)
+	{
+		SecurityKey securityKey = getSecurityKey(context);
+		return securityKey != null ? securityKey.encrypt(token) : null;
```

Comment:
Don't assume a ``null`` value from ``getSecurityKey``. The caller cannot do anything useful with it, as we already return ``null`` when the ``plaintext`` is ``null``.